app/build.gradle

```diff
@@ -1,13 +1,13 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.2"
+    compileSdkVersion 25
+    buildToolsVersion '25.0.0'
 
     defaultConfig {
         applicationId "org.loklak.android.wok"
-        minSdkVersion 11
-        targetSdkVersion 23
+        minSdkVersion 14
```

Comment:
But I read somewhere the up to 11 material design library is backward compatibile